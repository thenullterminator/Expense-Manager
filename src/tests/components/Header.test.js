import React from 'react';
import {shallow} from 'enzyme';
import Header from '../../components/NavBar';
import { exportAllDeclaration } from '@babel/types';

test('should render Header correctly',()=>{

      const wrapper=shallow(<Header />);
      expect(wrapper).toMatchSnapshot();

});