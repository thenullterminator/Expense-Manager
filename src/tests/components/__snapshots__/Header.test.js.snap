// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <p>
     
    <NavLink
      activeStyle={
        Object {
          "color": "red",
        }
      }
      exact={true}
      to="/"
    >
      home
    </NavLink>
  </p>
  <p>
     
    <NavLink
      activeStyle={
        Object {
          "color": "red",
        }
      }
      to="/create"
    >
      create
    </NavLink>
  </p>
  <p>
     
    <NavLink
      activeStyle={
        Object {
          "color": "red",
        }
      }
      to="/help"
    >
      help
    </NavLink>
  </p>
</div>
`;
